scripts/pre_commit_linter.py

```diff
@@ -788,6 +788,31 @@ def handle_starttag(self, tag, attrs):
         expected_indentation = self.indentation_level * self.indentation_width
         tag_line = self.file_lines[line_number - 1].lstrip()
         opening_tag = '<' + tag
+
+        # Check the indentation for content of style tag.
+        if tag_line.startswith(opening_tag) and tag == 'style':
+            next_line_number = line_number + 1
+            next_line = self.file_lines[next_line_number - 1]
+            next_line_expected_indentation = (
+                self.indentation_level + 1) * self.indentation_width
+            next_line_column_number = 0
+            # Check column number.
+            for i in next_line.split(' '):
```

Comment:
Instead of doing this, try this instead: https://stackoverflow.com/a/13649013

Reply:
yes. sure

Reply:
done